src/components/Modal/node/SendMessageModal.tsx

```diff
@@ -39,16 +39,17 @@ export const SendMessageModal = ({ peerId }: SendMessageModalProps) => {
   const [loader, set_loader] = useState<boolean>(false);
   const [status, set_status] = useState<string>('');
   const [numberOfHops, set_numberOfHops] = useState<number | ''>('');
-  const [sendMode, set_sendMode] = useState<'path' | 'automaticPath' | 'numberOfHops'>('automaticPath');
-  const [automaticPath, set_automaticPath] = useState<boolean>(true);
+  const [sendMode, set_sendMode] = useState<'path' | 'automaticPath' | 'numberOfHops' | 'directMessage'>('directMessage');
+  const [automaticPath, set_automaticPath] = useState<boolean>(false);
+  const [directMessage, set_directMessage] = useState<boolean>(true);
   const [message, set_message] = useState<string>('');
   const [receiver, set_receiver] = useState<string>(peerId ? peerId : '');
   const [openModal, set_openModal] = useState<boolean>(false);
 
   const maxLength = 500;
   const remainingChars = maxLength - message.length;
 
-  const nonAutomaticPathTooltip = 'Disable `automatic path` to enable `Number of hops`';
+  const nonAutomaticPathTooltip = 'Disable `direct message` and `automatic path` to enable this field';
```

Comment:
when direct message is enabled, this tooltip does not show up

Reply:
Thx, updated